src/main/java/org/example/C02_20_MultipleOfNumbers.java

```diff
@@ -2,13 +2,13 @@
 
 import java.util.Scanner;
 
-public class C02_20_MultipleOf {
+public class C02_20_MultipleOfNumbers {
```

Comment:
The name is still to vague. Try something more specific like NumberDividedBy3Or7.

Reply:
done